test/services/dumpers/agent-nodejs/agent-nodejs-dumper-mongoose-models.test.ts

```diff
@@ -153,4 +137,41 @@ describe('services > dumpers > agentNodejsDumper > mongoose models', () => {
       });
     });
   });
+
+  describe('dumping export', () => {
+    describe('javascript', () => {
+      it('should export expected values', async () => {
+        expect.assertions(2);
+
+        const language = languages.Javascript;
+
+        await dump(language, simple);
+
+        const generatedFile = fs.readFileSync(
+          `${appRoot}/test-output/${language.name}/mongodb/models/films.${language.fileExtension}`,
+          'utf8',
+        );
+
+        expect(generatedFile).toContain(`collectionName: 'films'`);
+        expect(generatedFile).toContain(`modelName: 'films'`);
+      });
+    });
+
+    describe('typescript', () => {
+      it('should export expected values', async () => {
+        expect.assertions(1);
+
+        const language = languages.Typescript;
+
+        await dump(language, simple);
+
+        const generatedFile = fs.readFileSync(
+          `${appRoot}/test-output/${language.name}/mongodb/models/films.${language.fileExtension}`,
+          'utf8',
+        );
+
+        expect(generatedFile).toContain(`export { FilmsInterface, filmsSchema }`);
+      });
+    });
+  });
```

Comment:
I guess those can't be included in the loop right ?

Reply:
Actually, in the future we could change the way index.js file is generated, to make it close to the index.ts one, but for now it is out of scope